libs/ha/ha_common_libs_k8s.py

```diff
@@ -1197,3 +1198,114 @@ def get_config_value(pod_obj, pod_list=None):
         data = yaml.safe_load(conf_fd)
 
         return True, data
+
+    @staticmethod
+    def setup_mock_monitor(node_obj):
+        """
+        :param node_obj: Object for node
+        :return: Bool
+        """
+        # TODO: Revisit once RPM for mock_health_event_publisher is available
+        LOGGER.info("Get HA pod name for setting up mock monitor")
+        ha_pod = node_obj.get_all_pods(pod_prefix=common_const.HA_POD_NAME_PREFIX)[0]
```

Comment:
Do we need to check running pod ?

Reply:
This will be called in setup_method so assuming, cluster will be in good state only.